src/core/navigation/Navigation.tsx

```diff
@@ -44,10 +73,17 @@ export function Navigation({
     utils: { getRoutedInternalHref },
   } = useConfig();
 
-  const currentLanguage = languages?.find(
-    (language) =>
-      language.code?.toLowerCase() === currentLanguageCode?.toLowerCase(),
-  );
+  // Language selection is required
+  if (!languages || !languages.length) {
+    return null;
+  }
+
+  const findLanguage = (languageCode: string) =>
+    languages.find(
+      (language) => language.code?.toLowerCase() === languageCode.toLowerCase(),
+    );
```

Comment:
Would be better moved away from the component to make the actual component cleaner. It's usually a bad practise to have functions inside functions: they are not reusable then and they become statefull instead of stateless. The component declares the function everytime the component is rerendered.

Reply:
Moved findLanguage to own function in [force push](https://github.com/City-of-Helsinki/react-helsinki-headless-cms/compare/294a6c11296c0e6f501307c48ae623543f26fafc..5ee8afd90130e2ef4543967a1a1afb70a72124cb)